ao.php

```diff
@@ -1,4 +1,4 @@
-<?php
+ <?php
```

Comment:
@Edzelopez looks like your editor issue at it again

Reply:
Yup, I figured what was wrong and corrected it.